pkgs/by-name/ti/tigervnc/package.nix

```
@@ -25,25 +24,17 @@
   ffmpeg,
 }:
 
-stdenv.mkDerivation rec {
-  version = "1.14.0";
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.15.0";
   pname = "tigervnc";
 
   src = fetchFromGitHub {
     owner = "TigerVNC";
     repo = "tigervnc";
-    rev = "v${version}";
-    sha256 = "sha256-TgVV/4MRsQHYKpDf9L5eHMLVpdwvNy1KPDIe7xMlQ9o=";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-ZuuvRJe/lAqULWObPxGHVJrDPCTK4IVSqX0K1rWOctw=";
```

```suggestion
    hash = "sha256-ZuuvRJe/lAqULWObPxGHVJrDPCTK4IVSqX0K1rWOctw=";
```

if we're already modernizing here

---

You're right, @Defelo !  Done and branch got updated.